SSpi ModulData     # Все данные по обмену полученные из Modul
SSpi DataModul     # Все данный по обмену отправленные из Data в Modul
SSpi DriverData    # Все данные по обмену полученные из Driver
SSpi DataDriver    # Все данный по обмену отправленные из Data в Driver